### PR TITLE
SDKs test build with Webrtc-83

### PR DESCRIPTION
**Pull request by piyushtank**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: VideoApp/Video-InternalTests/Specs/PasscodeComponentsSpec.swift [hunk 1]
```diff
@@ -100,11 +100,11 @@ class PasscodeComponentsSpec: QuickSpec {
                 }
 
             }
-            
+
             context("when string is invalid") {
                 context("when string length is 6") {
                     it("throws passcodeIncorrect error") {
-                        expect { try PasscodeComponents(string: "256984") }.to(throwError(AuthError.passcodeIncorrect))
+                        expect(expression: { try PasscodeComponents(string: "256984") }).to(throwError(AuthError.passcodeIncorrect))
```
Comment (timrozum) [inline VideoApp/Video-InternalTests/Specs/PasscodeComponentsSpec.swift:107]: This is a workaround for what seems to be a Swift bug. 

Here is a Nimble issue that is tracking it: https://github.com/Quick/Nimble/issues/809

Lets keep an eye on this issue. Hopefully it is fixed in a new beta and we can revert this change.

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods'
+gem 'cocoapods', '1.10.0.beta.2'
```
Comment (timrozum) [inline Gemfile:3]: Did this have to be hard-coded here to use the beta? `bundle update cocoapods` maybe only works for GA huh?

File: VideoApp/Video-CommunityTests/Info.plist [hunk 3]
```diff
@@ -17,6 +17,6 @@
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
-	<string>64</string>
+	<string>65</string>
```
Comment (timrozum) [inline VideoApp/Video-CommunityTests/Info.plist:20]: Probably not a problem but I'm curious why bundle version needed to be bumped?

Comment (paynerc) [reply VideoApp/Video-CommunityTests/Info.plist:20]: I'm not sure how this got bumped? If 64 is the proper version, we can reset it.

File: Gemfile (hunk 2 above)
Comment (paynerc) [reply Gemfile:3]: Yes, when using non SerVer variants you need to specify it directly.

File: VideoApp/Video-CommunityTests/Info.plist (hunk 3 above)
Comment (timrozum) [reply VideoApp/Video-CommunityTests/Info.plist:20]: I would say reset it to 64. When master is merged in it will automatically update to correct version.

Comment (paynerc) [reply VideoApp/Video-CommunityTests/Info.plist:20]: Master was merged into this branch yesterday?

Comment (timrozum) [reply VideoApp/Video-CommunityTests/Info.plist:20]: Master has 65. Hmmm that is strange. Is the `sdk-tests` base branch basically being used as a feature branch?

Comment (paynerc) [reply VideoApp/Video-CommunityTests/Info.plist:20]: That I'm not sure. I think we *should* create a `feature/4.0` feature branch and that is where we should be merging this work into.
